packages/v-pool/package.json

```diff
@@ -32,10 +32,9 @@
     "co": "4.6.0",
     "expect.js": "0.3.1",
     "lodash": "^4.17.11",
-    "mocha": "^7.1.2",
-    "pg-cursor": "^1.3.0"
```

Comment:
Another surprising change given the goal was just to change the version numbers.   Can you explain?   I suspect this is an unnecessary dependency, but want to make sure we discuss.

Reply:
pg-cursor was just another dependency that we have never been using but somehow slipped through during initial cleanup